src/Integration.Vsix/Manifests/VS2019/source.extension.vsixmanifest

```diff
@@ -0,0 +1,38 @@
+        <Identity Id="SonarLint.b986f788-6a16-4a3a-a68b-c757f6b1b7d5" Version="4.9.0.0" Language="en-US" Publisher="SonarSource SA" />
+        <DisplayName>SonarLint for Visual Studio 2019</DisplayName>
+        <Description xml:space="preserve">Roslyn based static code analysis: Find and instantly fix nasty bugs and code smells in C#, VB.Net, C, C++ and JS.</Description>
+        <MoreInfo>http://vs.sonarlint.org</MoreInfo>
+        <License>LICENSE</License>
+        <GettingStartedGuide>http://vs.sonarlint.org</GettingStartedGuide>
+        <Icon>Resources\sonarlint_wave_128px.png</Icon>
+        <PreviewImage>Resources\sonarlint_200.png</PreviewImage>
+        <Tags>SonarLint, SonarQube, Analysis, Roslyn, CodeAnalysis, Analyzer, Code analysis, Sonar, Debt, Technical, Tech, Quality</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)" />
+    </Installation>
+    <Dependencies>
+        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.6,)" />
+    </Dependencies>
+    <Assets>
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="Integration" Path="|Integration|" />
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%|" />
+        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="Progress" Path="|Progress|" AssemblyName="|Progress;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="ProgressVS" Path="|ProgressVS|" AssemblyName="|ProgressVS;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="SonarAnalyzer.dll" />
+        <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="SonarAnalyzer.CSharp.dll" />
+        <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="SonarAnalyzer.VisualBasic.dll" />
+        <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="Google.Protobuf.dll" />
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="Integration.TeamExplorer" Path="|Integration.TeamExplorer|" />
+    </Assets>
+    <Prerequisites>
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.Compiler" Version="[16.0,17.0)" DisplayName="C# and Visual Basic Roslyn compilers" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.Static.Analysis.Tools" Version="[16.0,17.0)" DisplayName="Static analysis tools" />
```

Comment:
I pulled down the code and it built ok, but I could install the vsix it produced. It gave a message about one of the pre-requisites not being available (although it didn't say which one).